lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -321,7 +321,7 @@ def test_anonymous_get_xblock_callback(self):
         """Test that anonymous GET is allowed."""
         dispatch_url = self._get_dispatch_url()
         response = self.client.get(dispatch_url)
-        self.assertEquals(200, response.status_code)
+        self.assertEquals(403, response.status_code)
```

Comment:
Whenever a change is made to a test, we need to be aware that we are potentially breaking compatibility or a feature.  In this case, this test is ensuring that _"anonymous GET is allowed"_.  And now the test is showing that _"anonymous GET is **not** allowed"_.